_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and machine learning (ML) tools for OpenSearch. It provides data analysts with the following abilities:
+
+- Call OpenSearch indexes and manipulate each index as a [pandas.DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html). These data frames support complex filtering and aggregation operations.
+- Provides basic support to upload models into OpenSearch using the [ML Commons plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index/)
```

Comment:
`upload sentence transformer models` will more precise for now I think.

Reply:
In addition, we also support `model training/tuning` for SentenceTransformer model. @mingshl you wanna add anything here?

Reply:
line 10, we should also add that  we now support customers to use their synthetic queries to fine tune and train sentence transformer models.